op-bindings/Makefile

```diff
@@ -1,9 +1,11 @@
 SHELL := /usr/bin/env bash
 
 pkg := bindings
+pkg-preview := bindingspreview
```

Comment:
What does the preview do? Is it just a copy-paste of `pkg` to expose `OptimismPortal2`?

Reply:
Yeah it's just a different package because `OptimismPortal2` conflicts with `OptimismPortal` and Go was mad about it. I figure we can use this package for any other preview contracts that might conflict.